docs/guides/soloproject/soloproject.md

```diff
@@ -67,7 +68,7 @@ intermediate (Tier 2), or experienced (Tier 3).
 
 <p>Your Solo Project <strong><em>must</em></strong> be defined using CSS and HTML, which also uses logic (designed and written by you) which dynamically processes events and inputs to display or modify information on the screen.</p>
 
-<p>In addition, Tier 1 Chingus will only use vanilla CSS/HTML + a programming language for Solo Projects and Voyages. Those using any form of FE framework such as React, VueJS, Svelte, etc. should apply for Tier 2 or higher.</p>
+<p>In addition, Tier 1 Chingus will only use a programming language for Solo Projects and Voyages, also using a FE framework such as React, VueJS, Svelte, etc. If you haven't yet learned a FE framework we you will need to learn that skill before submitting a Solo Project and joining a Voyage.</p>
```

Comment:
What about something like this?

Tier 1 Chingus will use a programming language in addition to a framework such as React, Vue, Svelte, etc. If you haven't yet learned a FE framework, you will need to learn that skill before submitting a Solo Project and joining a Voyage. We recommend Scrimba <insert affiliate link>

Reply:
I incorporated this suggestion and linked it to the Partners page in the Handbook. Thanks!